app/src/renderer/components/common/ShortBech32.vue

```diff
@@ -25,7 +24,9 @@ export default {
     }
   },
   data: () => ({
-    showSuccess: false
+    showSuccess: false,
+    /* istanbul ignore next */
```

Comment:
istanbul?

Reply:
istanbul is responsible for code coverage. this tells it to exclude the following statement from the coverage report